### PR TITLE
Added release notes

### PR DESCRIPTION
**Pull request by borgmanJeremy**

### CODE REVIEW

**Review by hosiet — CHANGES_REQUESTED** [review]
I believe there are several grammar errors in the "Known Issues" part. Please consider fixing them.

Besides, it's worthwhile to point out the actual format of "generic packages" by listing out their names (AppImage, Snap, etc.).

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+![](images/counter.gif)
+
+
+### Sidebar
+A button has been added to open the sidebar. This tool was previously only accessible by hitting "Spacebar" which made usage on tablets difficult. This sidebar allows advanced modifications for many tools. 
+
+![](images/sidebar.gif)
+
+### Sidebar
+A button has been added to open the sidebar. This tool was previously only accessible by hitting "Spacebar" which made usage on tablets difficult. This sidebar allows advanced modifications for many tools.
```
Comment (mmahmoudian) [inline docs/ReleaseNotes.md:52]: This looks like to be a duplicate of lines 46 and 47

**Review by mmahmoudian — CHANGES_REQUESTED** [review]

**Review by borgmanJeremy — COMMENTED** [review]
Sorry I didnt see you made the edits directly until I had already pushed the fix.

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# 0.8 Release Notes
+Thanks to all the testers and contributors that helped make version 0.8! We are very excited to have improved many bugs and added new features in version 0.8. 
+
+## Known Issues
+* Wayland support is experimental. In generic packages (Snap, Flatpak, AppImage) there may be extra issues with Wayland. 
+* In generic packages(Snap, Flatpak, AppImage) due to confinement "Open With External Application" does not work.
```
Comment (ZetaoYang) [inline docs/ReleaseNotes.md:6]: AppImage "Open With External Application" works fine, reference https://github.com/flameshot-org/flameshot/issues/933#issuecomment-694779370.

File: docs/ReleaseNotes.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# 0.8 Release Notes
+Thanks to all the testers and contributors that helped make version 0.8! We are very excited to have improved many bugs and added new features in version 0.8. 
+
+## Known Issues
+* Wayland support is experimental. In generic packages (Snap, Flatpak, AppImage) there may be extra issues with Wayland. 
+* In generic packages(Snap, Flatpak, AppImage) due to confinement "Open With External Application" does not work. 
+* If close after capture is enabled, and a user copies the image to clipboard, the image may not actually be in the clipboard. We recommend using the "Save" feature with close after capture.
```
Comment (ZetaoYang) [inline docs/ReleaseNotes.md:7]: I suggest putting `close after capture` in quotation marks. -> "close after capture"

File: docs/ReleaseNotes.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+## New Features
+* Fix capture after pressing ctrl + S during textarea input (#311)
+* Add translation:
+  - Japanese (#361)
+  - Brazilian Portuguese (#364)
+  - Serbian (#376)
+  - Dutch (#440)
+  - Ukrainian (#464)
+  - German (#467)
+  - Slovak (#525)
```
Comment (ZetaoYang) [inline docs/ReleaseNotes.md:19]: Some languages are not mentioned here yet, so please update it according to this list https://github.com/flameshot-org/translation-instruction#status .

**Review by ZetaoYang — CHANGES_REQUESTED** [review]
Please check out my two comments.

**Review by hosiet — APPROVED** [review]
Now this looks okay to me.

**Review by ZetaoYang — APPROVED** [review]
Now it's ok to me.

**Pull request merged**